src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/script.js.template

```diff
@@ -0,0 +1,481 @@
+/*
```

Comment:
I note that at least in part this is a rename of `script.js` (and rightly so) that Git has failed to detect (grrr.) Others may want to use external tools to compare the old and new forms. The differences are primarily a significant expansion of these lines in the old code:

```
// Dynamically set scroll margin to accomodate for draft header
document.addEventListener("DOMContentLoaded", function(e) {
    document.querySelectorAll(':not(input)[id]').forEach(
        function(c) {
            c.style["scroll-margin-top"] = Math.ceil(document.querySelector("header").offsetHeight) + "px"
        });
});
```

Reply:
While doing rename and edit in separate commits may help the PR, it does not help the long term code archaeology, because the commits will be squashed when integrating them to the mainline repo. The rename-detection is a regrettable shortcoming of Git itself; there is nothing we can do about it.